cpp/tests/test_abm_person.cpp

```diff
@@ -222,75 +226,79 @@ TEST(TestPerson, interact)
 TEST(TestPerson, applyMaskIntervention)
 {
     auto rng = mio::RandomNumberGenerator();
+    auto t   = mio::abm::TimePoint(0);
 
     mio::abm::Location home(mio::abm::LocationType::Home, 0, num_age_groups);
     mio::abm::Location target(mio::abm::LocationType::Work, 0, num_age_groups);
```

Comment:
The target is being tested again in this test as the function `apply_mask_intervention` doesn't exist anymore.
Also, since we have the test for is_compliant now, I think we can remove this entire test.

Reply:
Yes, I now deleted this test.